cli/Dockerfile

```diff
@@ -18,8 +18,6 @@ RUN export DEBIAN_FRONTEND=noninteractive \
      && curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | gpg --dearmor -o /usr/share/keyrings/githubcli-archive-keyring.gpg \
      && echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | tee /etc/apt/sources.list.d/github-cli.list > /dev/null \
      && apt-get update && apt-get install -y gh \ 
-     # az cli
-     && curl -sSL https://aka.ms/InstallAzureCLIDeb | bash \
```

Comment:
We should keep this for the Azure DevOps pipelines for now due to #1158

Reply:
Good catch, I did not realize that the we ran the the pipeline inside this container.